src/androidTest/java/com/nutomic/syncthingandroid/test/syncthing/RestApiTest.java

```diff
@@ -82,8 +80,8 @@ public void testReadableFileSize() {
     }
     
     public void testGetReadableTransferRate() {
-        assertEquals("1 MiB/s", RestApi.readableTransferRate(getContext(), 8388608L));
-        assertEquals("1 GiB/s", RestApi.readableTransferRate(getContext(), 8589934592L));
+        assertEquals("1 Mb/s", RestApi.readableTransferRate(getContext(), 8388608L));
+        assertEquals("1 Gb/s", RestApi.readableTransferRate(getContext(), 8589934592L));
```

Comment:
So what shall I do then?
Change my changes back? Also remove the test? Or leave it like it is?